server/Dockerfile

```diff
@@ -49,6 +49,12 @@ COPY log4j2.properties .
 COPY database.properties.tmpl .
 COPY docker-entrypoint.sh .
 
+RUN groupadd -r --gid 201 scalardb && \
+    useradd -r --uid 201 -g scalardb scalardb
+RUN chown -R scalardb:scalardb /scalardb/server
```

Comment:
Change owner of files under /scalardb/server because each file may be read at runtime.